arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-04-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The list of property names to be returned. You can save bandwith by selecting only the properties you need."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get a page of digest events",
+            "schema": {
+              "$ref": "#/definitions/EventDataCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/EventData"
+      }
+    },
+    "/providers/microsoft.insights/eventtypes/management/digestEvents": {
+      "get": {
+        "tags": [
+          "Event"
+        ],
+        "operationId": "TenantDigestEvent_List",
+        "description": "The List Tenant Digest Event Values operation lists the tenant-level digest events.",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-04-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The filter to apply on the operation."
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The list of property names to be returned. You can save bandwith by selecting only the properties you need."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get a page of tenant digest events",
+            "schema": {
+              "$ref": "#/definitions/EventDataCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "#/definitions/EventData"
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/microsoft.insights/eventtypes/management/values": {
+      "get": {
+        "tags": [
+          "Event"
+        ],
+        "operationId": "Event_List",
```

Comment:
- DigestEvent_List
- TenantDigestEvent_List
- Event_List
- TenantEvent_List

All these operations return the same response Model EvenDataCollection which is an Array of EventData model definition.

So what is the difference between Event and DigestEvent?